Files lucene-1.9.1/build/classes/java/org/apache/lucene/LucenePackage.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/LucenePackage.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/Analyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/Analyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/CharTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/CharTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/ISOLatin1AccentFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/ISOLatin1AccentFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/KeywordAnalyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/KeywordAnalyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/KeywordTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/KeywordTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/LengthFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/LengthFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/LetterTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/LetterTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/LowerCaseFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/LowerCaseFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/LowerCaseTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/LowerCaseTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/PerFieldAnalyzerWrapper.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/PerFieldAnalyzerWrapper.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/PorterStemFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/PorterStemFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/PorterStemmer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/PorterStemmer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/SimpleAnalyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/SimpleAnalyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/StopAnalyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/StopAnalyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/StopFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/StopFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/Token.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/Token.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/TokenFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/TokenFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/TokenStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/TokenStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/Tokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/Tokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/WhitespaceAnalyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/WhitespaceAnalyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/WhitespaceTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/WhitespaceTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/WordlistLoader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/WordlistLoader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/de/WordlistLoader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/de/WordlistLoader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/CharStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/CharStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/FastCharStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/FastCharStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/ParseException.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/ParseException.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/StandardAnalyzer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/StandardAnalyzer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/StandardFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/StandardFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizerConstants.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizerConstants.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizerTokenManager.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/StandardTokenizerTokenManager.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/Token.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/Token.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/analysis/standard/TokenMgrError.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/analysis/standard/TokenMgrError.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/DateField.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/DateField.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/DateTools$Resolution.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/DateTools$Resolution.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/DateTools.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/DateTools.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/Document.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/Document.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/Field$Index.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/Field$Index.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/Field$Store.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/Field$Store.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/Field$TermVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/Field$TermVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/Field.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/Field.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/document/NumberTools.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/document/NumberTools.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileReader$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileReader$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileReader$CSIndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileReader$CSIndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileReader$FileEntry.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileReader$FileEntry.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileWriter$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileWriter$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileWriter$FileEntry.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileWriter$FileEntry.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/CompoundFileWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/CompoundFileWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/DocumentWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/DocumentWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FieldInfo.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FieldInfo.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FieldInfos.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FieldInfos.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FieldsReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FieldsReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FieldsWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FieldsWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FilterIndexReader$FilterTermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/FilterIndexReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/FilterIndexReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexFileNameFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexFileNameFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexFileNames.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexFileNames.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexModifier.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexModifier.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexReader$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexReader$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexReader$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexReader$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexReader$FieldOption.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexReader$FieldOption.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter$3.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter$3.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter$4.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter$4.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter$5.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter$5.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/IndexWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/IndexWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultiReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultiReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultiTermDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultiTermDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultiTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultiTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultiTermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultiTermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultipleTermPositions$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultipleTermPositions$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultipleTermPositions$IntQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultipleTermPositions$IntQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultipleTermPositions$TermPositionsQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultipleTermPositions$TermPositionsQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/MultipleTermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/MultipleTermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/ParallelReader$ParallelTermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/ParallelReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/ParallelReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/Posting.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/Posting.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentInfo.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentInfo.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentInfos.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentInfos.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentMergeInfo.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentMergeInfo.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentMergeQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentMergeQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentMerger.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentMerger.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentReader$Norm.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentReader$Norm.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentTermDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentTermDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentTermPositionVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentTermPositionVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentTermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentTermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/SegmentTermVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/SegmentTermVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/Term.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/Term.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermBuffer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermBuffer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermFreqVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermFreqVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermInfo.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermInfo.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermInfosReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermInfosReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermInfosWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermInfosWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermPositionVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermPositionVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermPositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermPositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorOffsetInfo.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorOffsetInfo.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorsReader.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorsReader.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorsWriter$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorsWriter$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorsWriter$TVField.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorsWriter$TVField.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorsWriter$TVTerm.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorsWriter$TVTerm.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/index/TermVectorsWriter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/index/TermVectorsWriter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/CharStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/CharStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/FastCharStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/FastCharStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/MultiFieldQueryParser.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/MultiFieldQueryParser.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/ParseException.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/ParseException.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParser$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParser$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParser$JJCalls.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParser$JJCalls.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParser$LookaheadSuccess.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParser$LookaheadSuccess.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParser$Operator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParser$Operator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParser.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParser.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParserConstants.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParserConstants.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/QueryParserTokenManager.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/QueryParserTokenManager.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/Token.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/Token.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/queryParser/TokenMgrError.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/queryParser/TokenMgrError.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanClause$Occur.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanClause$Occur.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanClause.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanClause.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanQuery$BooleanWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanQuery$BooleanWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanQuery$BooleanWeight2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanQuery$BooleanWeight2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanQuery$TooManyClauses.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanQuery$TooManyClauses.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer$Bucket.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer$Bucket.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer$BucketTable.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer$BucketTable.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer$Collector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer$Collector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer$SubScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer$SubScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer2$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer2$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer2$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer2$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer2$Coordinator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer2$Coordinator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer2$SingleMatchScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer2$SingleMatchScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/BooleanScorer2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/BooleanScorer2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/CachingWrapperFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/CachingWrapperFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConjunctionScorer$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConjunctionScorer$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConjunctionScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConjunctionScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConstantScoreQuery$ConstantScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConstantScoreQuery$ConstantScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConstantScoreQuery$ConstantWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConstantScoreQuery$ConstantWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConstantScoreQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConstantScoreQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ConstantScoreRangeQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ConstantScoreRangeQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DateFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DateFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DefaultSimilarity.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DefaultSimilarity.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DisjunctionMaxQuery$DisjunctionMaxWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DisjunctionMaxQuery$DisjunctionMaxWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DisjunctionMaxQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DisjunctionMaxQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DisjunctionMaxScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DisjunctionMaxScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DisjunctionSumScorer$ScorerQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DisjunctionSumScorer$ScorerQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/DisjunctionSumScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/DisjunctionSumScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ExactPhraseScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ExactPhraseScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Explanation.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Explanation.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCache$FloatParser.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCache$FloatParser.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCache$IntParser.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCache$IntParser.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCache$StringIndex.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCache$StringIndex.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCache.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCache.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCacheImpl$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCacheImpl$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCacheImpl$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCacheImpl$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCacheImpl$Entry.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCacheImpl$Entry.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldCacheImpl.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldCacheImpl.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldDoc.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldDoc.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldDocSortedHitQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldDocSortedHitQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$3.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$3.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$4.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue$4.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FieldSortedHitQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Filter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Filter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FilteredQuery$1$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FilteredQuery$1$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FilteredQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FilteredQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FilteredQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FilteredQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FilteredTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FilteredTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FuzzyQuery$ScoreTerm.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FuzzyQuery$ScoreTerm.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FuzzyQuery$ScoreTermQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FuzzyQuery$ScoreTermQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FuzzyQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FuzzyQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/FuzzyTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/FuzzyTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Hit.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Hit.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/HitCollector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/HitCollector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/HitDoc.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/HitDoc.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/HitIterator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/HitIterator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/HitQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/HitQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Hits.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Hits.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/IndexSearcher$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/IndexSearcher$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/IndexSearcher.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/IndexSearcher.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery$MatchAllDocsWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery$MatchAllDocsWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery$MatchAllScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery$MatchAllScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MatchAllDocsQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiPhraseQuery$MultiPhraseWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiPhraseQuery$MultiPhraseWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiPhraseQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiPhraseQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiSearcher$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiSearcher$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiSearcher$CachedDfSource.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiSearcher$CachedDfSource.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiSearcher.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiSearcher.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiSearcherThread.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiSearcherThread.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/MultiTermQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/MultiTermQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/NonMatchingScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/NonMatchingScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ParallelMultiSearcher$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ParallelMultiSearcher$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ParallelMultiSearcher.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ParallelMultiSearcher.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhrasePositions.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhrasePositions.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhrasePrefixQuery$PhrasePrefixWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhrasePrefixQuery$PhrasePrefixWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhrasePrefixQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhrasePrefixQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhraseQuery$PhraseWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhraseQuery$PhraseWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhraseQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhraseQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhraseQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhraseQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PhraseScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PhraseScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/PrefixQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/PrefixQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Query.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Query.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/QueryFilter$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/QueryFilter$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/QueryFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/QueryFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/QueryTermVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/QueryTermVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/RangeFilter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/RangeFilter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/RangeQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/RangeQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/RemoteSearchable.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/RemoteSearchable.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/RemoteSearchable_Stub.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/RemoteSearchable_Stub.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ReqExclScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ReqExclScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ReqOptSumScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ReqOptSumScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ScoreDoc.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ScoreDoc.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ScoreDocComparator$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ScoreDocComparator$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ScoreDocComparator$2.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ScoreDocComparator$2.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/ScoreDocComparator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/ScoreDocComparator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Scorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Scorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Searchable.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Searchable.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Searcher.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Searcher.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Similarity.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Similarity.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SimilarityDelegator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SimilarityDelegator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SloppyPhraseScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SloppyPhraseScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Sort.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Sort.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SortComparator$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SortComparator$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SortComparator.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SortComparator.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SortComparatorSource.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SortComparatorSource.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/SortField.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/SortField.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TermQuery$TermWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TermQuery$TermWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TermQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TermQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TermScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TermScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TopDocCollector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TopDocCollector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TopDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TopDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TopFieldDocCollector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TopFieldDocCollector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/TopFieldDocs.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/TopFieldDocs.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/Weight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/Weight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/WildcardQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/WildcardQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/WildcardTermEnum.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/WildcardTermEnum.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/NearSpans$CellQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/NearSpans$CellQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/NearSpans$SpansCell.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/NearSpans$SpansCell.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/NearSpans.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/NearSpans.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanFirstQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanFirstQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanFirstQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanFirstQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanNearQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanNearQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanNotQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanNotQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanNotQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanNotQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery$SpanQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery$SpanQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanOrQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanScorer.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanScorer.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanTermQuery$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanTermQuery$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanTermQuery.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanTermQuery.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/SpanWeight.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/SpanWeight.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/search/spans/Spans.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/search/spans/Spans.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/BufferedIndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/BufferedIndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/BufferedIndexOutput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/BufferedIndexOutput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/Directory.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/Directory.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/FSDirectory$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/FSDirectory$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/FSDirectory.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/FSDirectory.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/FSIndexInput$Descriptor.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/FSIndexInput$Descriptor.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/FSIndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/FSIndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/FSIndexOutput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/FSIndexOutput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/IndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/IndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/IndexOutput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/IndexOutput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/InputStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/InputStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/Lock$With.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/Lock$With.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/Lock.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/Lock.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/MMapDirectory$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/MMapDirectory$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/MMapDirectory$MMapIndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/MMapDirectory$MMapIndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/MMapDirectory$MultiMMapIndexInput.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/MMapDirectory$MultiMMapIndexInput.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/MMapDirectory.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/MMapDirectory.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/OutputStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/OutputStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/RAMDirectory$1.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/RAMDirectory$1.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/RAMDirectory.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/RAMDirectory.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/RAMFile.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/RAMFile.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/RAMInputStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/RAMInputStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/store/RAMOutputStream.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/store/RAMOutputStream.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/BitVector.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/BitVector.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/Constants.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/Constants.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/FixedThreadLocal.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/FixedThreadLocal.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/Parameter.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/Parameter.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/PriorityQueue.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/PriorityQueue.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/SmallFloat.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/SmallFloat.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/StringHelper.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/StringHelper.class differ
Files lucene-1.9.1/build/classes/java/org/apache/lucene/util/ToStringUtils.class and lucene-1.9.1-protrade/build/classes/java/org/apache/lucene/util/ToStringUtils.class differ
Files lucene-1.9.1/build/lucene-core-1.9.2-protrade.jar and lucene-1.9.1-protrade/build/lucene-core-1.9.2-protrade.jar differ
diff -N -r -u lucene-1.9.1/common-build.xml lucene-1.9.1-protrade/common-build.xml
--- lucene-1.9.1/common-build.xml	2006-03-02 12:29:01.000000000 -0800
+++ lucene-1.9.1-protrade/common-build.xml	2006-05-11 10:08:57.000000000 -0700
@@ -22,7 +22,7 @@
 
   <property name="name" value="${ant.project.name}"/>
   <property name="Name" value="Lucene"/>
-  <property name="version" value="1.9.2-dev"/>
+  <property name="version" value="1.9.2-protrade"/>
   <property name="year" value="2000-${current.year}"/>
   <property name="final.name" value="lucene-${name}-${version}"/>
 
diff -N -r -u lucene-1.9.1/src/java/org/apache/lucene/index/SegmentReader.java lucene-1.9.1-protrade/src/java/org/apache/lucene/index/SegmentReader.java
--- lucene-1.9.1/src/java/org/apache/lucene/index/SegmentReader.java	2006-02-20 10:20:14.000000000 -0800
+++ lucene-1.9.1-protrade/src/java/org/apache/lucene/index/SegmentReader.java	2006-05-11 10:00:52.000000000 -0700
@@ -25,6 +25,7 @@
 import org.apache.lucene.store.IndexOutput;
 import org.apache.lucene.store.Directory;
 import org.apache.lucene.util.BitVector;
+import org.apache.lucene.util.FixedThreadLocal;
 import org.apache.lucene.search.DefaultSimilarity;
 
 /**
@@ -38,7 +39,7 @@
 
   TermInfosReader tis;
   TermVectorsReader termVectorsReaderOrig = null;
-  ThreadLocal termVectorsLocal = new ThreadLocal();
+  FixedThreadLocal termVectorsLocal = new FixedThreadLocal();
 
   BitVector deletedDocs = null;
   private boolean deletedDocsDirty = false;
@@ -161,12 +162,6 @@
     }
   }
 
-   protected void finalize() {
-     // patch for pre-1.4.2 JVMs, whose ThreadLocals leak
-     termVectorsLocal.set(null);
-     super.finalize();
-   }
-
   protected void doCommit() throws IOException {
     if (deletedDocsDirty) {               // re-write deleted
       deletedDocs.write(directory(), segment + ".tmp");
diff -N -r -u lucene-1.9.1/src/java/org/apache/lucene/index/TermInfosReader.java lucene-1.9.1-protrade/src/java/org/apache/lucene/index/TermInfosReader.java
--- lucene-1.9.1/src/java/org/apache/lucene/index/TermInfosReader.java	2006-02-20 10:20:14.000000000 -0800
+++ lucene-1.9.1-protrade/src/java/org/apache/lucene/index/TermInfosReader.java	2006-05-11 09:54:47.000000000 -0700
@@ -19,6 +19,7 @@
 import java.io.IOException;
 
 import org.apache.lucene.store.Directory;
+import org.apache.lucene.util.FixedThreadLocal;
 
 /** This stores a monotonically increasing set of <Term, TermInfo> pairs in a
  * Directory.  Pairs are accessed either by Term or by ordinal position the
@@ -29,7 +30,7 @@
   private String segment;
   private FieldInfos fieldInfos;
 
-  private ThreadLocal enumerators = new ThreadLocal();
+  private FixedThreadLocal enumerators = new FixedThreadLocal();
   private SegmentTermEnum origEnum;
   private long size;
 
@@ -54,11 +55,6 @@
 			  fieldInfos, true);
   }
 
-  protected void finalize() {
-    // patch for pre-1.4.2 JVMs, whose ThreadLocals leak
-    enumerators.set(null);
-  }
-
   public int getSkipInterval() {
     return origEnum.skipInterval;
   }
diff -N -r -u lucene-1.9.1/src/java/org/apache/lucene/util/FixedThreadLocal.java lucene-1.9.1-protrade/src/java/org/apache/lucene/util/FixedThreadLocal.java
--- lucene-1.9.1/src/java/org/apache/lucene/util/FixedThreadLocal.java	1969-12-31 16:00:00.000000000 -0800
+++ lucene-1.9.1-protrade/src/java/org/apache/lucene/util/FixedThreadLocal.java	2006-05-11 10:08:15.000000000 -0700
@@ -0,0 +1,33 @@
+package org.apache.lucene.util;
+
+import java.util.*;
+
+/**
+ * version of ThreadLocal that does not leak memory (ThreadLocal is fixed in JDK 1.5) 
+ */
+public class FixedThreadLocal {
+    private final Map values = new WeakHashMap();
+    
+    protected Object initialValue() {
+        return null;
+    }
+    
+    public final Object get() {
+        final Object key = Thread.currentThread();
+	synchronized ( values ) {
+            Object value = values.get(key);
+            if ( value == null ) {
+                value = initialValue();
+                values.put(key,value);
+            }
+            return value;
+        }
+    }
+    
+    public final synchronized void set(Object value) {
+        final Object key = Thread.currentThread();
+        synchronized ( values ) {
+            values.put(key,value);
+        }
+    }
+}
